src/database/database-postgresql.cpp

```diff
@@ -637,4 +642,174 @@ void PlayerDatabasePostgreSQL::listPlayers(std::vector<std::string> &res)
 	PQclear(results);
 }
 
+AuthDatabasePostgreSQL::AuthDatabasePostgreSQL(const std::string &connect_string) :
+		Database_PostgreSQL(connect_string), AuthDatabase()
+{
+	connectToDatabase();
+}
+
+void AuthDatabasePostgreSQL::createDatabase()
+{
+	createTableIfNotExists("auth",
+		"CREATE TABLE auth ("
+			"id SERIAL,"
+			"name VARCHAR(32) UNIQUE,"
```

Comment:
32 seems a bit short
In fact the [postgres documentation](https://www.postgresql.org/docs/current/datatype-character.html) suggests to just use `TEXT` in all cases.

Reply:
yep i prefer to use TEXT too, it was just to keep compat with sqlite3 format, but it's not a good pg practice, removing